go.mod

```diff
@@ -124,7 +124,11 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.1 // indirect
 )
 
-replace go.uber.org/atomic => github.com/uber-go/atomic v1.9.0
+replace (
+  github.com/kserve/kserve => github.com/kserve/kserve v0.10.0-rc0
```

Comment:
Thanks for your new changes @JasmondL

To fix the `missing go.sum entry` errors, please run `go mod tidy` and add/commit the updated `go.sum` file

Reply:
Done